server/app/views/HtmlBundle.java

```diff
@@ -224,6 +228,16 @@ private DivTag renderModals() {
             .withId("modal-container")
             .withClasses(BaseStyles.MODAL_CONTAINER)
             .with(div().withId("modal-glass-pane").withClasses(BaseStyles.MODAL_GLASS_PANE));
+    // Validate that only one modal has the "displayOnLoad" setting, since popping multiple
+    // modals doesn't make sense. A warning is logged rather than throwing since client-side
+    // code will choose the first modal with the attribute.
+    long displayOnLoadModalCount = modals.stream().filter(Modal::displayOnLoad).count();
+    if (displayOnLoadModalCount > 1) {
+      logger.warn(
+          String.format(
+              "Multiple modals found containing display on load. Got: %d",
```

Comment:
nit: "Got" reads weird

Suggestions:
- Mutliple (%d) modals
- %d modals
- Found %d

Reply:
Sorry, that's a reaction to my Go-programming where things tend to use a got/want convention for error messages. That definitely isn't the most readable here. Fixed.